flask/app/datasets.py

```diff
@@ -199,6 +199,9 @@ def list_datasets(page: int, limit: int) -> Response:
             "family_codename": dataset.tissue_sample.participant.family.family_codename,
             "created_by": dataset.created_by.username,
             "updated_by": dataset.updated_by.username,
+            "group_code": [group.group_code for group in dataset.groups]
```

Comment:
#582 mentions showing permissions groups in dialogs. Will this conditional prevent that functionality or do we want to display the group codes in the dialogs only if the dataset is associated with more than one group?

Reply:
I'm actually pretty unsure about this. It might just be fine to just always send group code information, and then just selectively hide it on the frontend. I don't see any security concerns for people knowing who else has access to the datasets that they have access to. @kevinlul thoughts?

Reply:
So `/api/groups` only allows regular users to read groups that they are a member of. This would make a user aware of other groups, though there would be ways to indirectly determine that anyway if changes occurred that were not because of any user in the group, and the username is revealed when it's edited. I think we are okay without the condition?